src/Subscription.ts

```diff
@@ -13,7 +13,7 @@ export type TeardownLogic = AnonymousSubscription | Function | void;
 
 export interface ISubscription extends AnonymousSubscription {
   unsubscribe(): void;
-  closed: boolean;
+  readonly closed: boolean;
```

Comment:
@david-driscoll check #2234 as well, this PR is subset of it only updated type definition, while it does updates actual code behaviors.